op-bindings/predeploys/addresses.go

```diff
@@ -46,6 +46,29 @@ var (
 	Predeploys = make(map[string]*common.Address)
 )
 
+// IsProxied returns true for predeploys that will sit behind a proxy contract
+func IsProxied(predeployAddr common.Address) bool {
+	switch predeployAddr {
+	case LegacyERC20ETHAddr:
```

Comment:
The false cases can be collapsed into a single branch

Reply:
You mean via fallthrough?

Reply:
Like this example:

```golang
func WhiteSpace(c rune) bool {
    switch c {
    case ' ', '\t', '\n', '\f', '\r':
        return true
    }
    return false
}
```

I believe you can also have something like:
```
case foo:
case bar:
    return true
```

Reply:
ah got it, don't need a fallthrough with a return statement outside the case. fixed.